jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -523,31 +520,6 @@ Browser: ${navigator.userAgent}`
     @param {Function} next Run the next handler.
   */
   _clearPostDeployment(state, next) {
-    const closeTime = new Date().getTime();
-
-    if (this.postDeploymentPanel
-      && this.postDeploymentPanel.openTime
-      && this.postDeploymentPanel.entityId) {
-      const entityId = this.postDeploymentPanel.entityId;
-      const openTime = this.postDeploymentPanel.openTime;
-      const action = 'Close post deployment panel';
-
-      // Round it to the nearest second.
-      let timeOpen = Math.round(
-        (closeTime - openTime) / 1000
-      );
-      let args = [
-        `${timeOpen}s`,
-        entityId
-      ];
-
-      this.sendAnalytics(
```

Comment:
Do we not care about these analytics anymore?

Reply:
Not at the moment, we need to find a better way of tracking these.